mlflow/server/js/src/reducers/Reducers.js

```diff
@@ -270,7 +283,7 @@ export const getSharedMetricKeysByRunUuids = (runUuids, state) =>
 
 export const getApis = (requestIds, state) => {
   return requestIds.map((id) => (
-    state.apis[id]
+    state.apis[id] || {}
```

Comment:
As we are moving fetchings out of `getDerivedStateFromProps`, this will address the NPE when we do not load before the first rendering.